frontend/packages/kubevirt-plugin/src/components/create-vm/create-vm.tsx

```diff
@@ -366,15 +379,33 @@ export const CreateVM: React.FC<RouteComponentProps> = ({ location }) => {
             {(footerProps) => (
               <Footer
                 {...footerProps}
-                formIsValid={state.isValid}
+                formIsValid={state.isValid && isValidSSHKey}
                 isCreating={isCreating}
                 createError={createError}
                 cleanError={() => setCreateError(undefined)}
                 onFinish={async () => {
                   resetError();
                   setCreating(true);
                   try {
-                    await createVM(state.template, sourceStatus, bootState, state, scConfigMap);
+                    const vm = await createVM(
```

Comment:
can we move the whole `onFinish` block outside of the JSX

Reply:
not really outside of jsx as we are getting data from hooks, but can be extracted to a function or a custom hook

Reply:
fine leave it